### DISCUSSION TITLE
No stack trace for "Failed to query actuator with error"

### DISCUSSION THREAD

**Discussion by prateekkarmakar** [specmatic/specmatic#1495] [Ask For Help]
In my project get the following error in the logs :

Failed to query actuator with error: null cannot be cast to non-null type io.specmatic.core.value.JSONArrayValue. 

This happens just after the specmatic.yaml file is loaded and the contract defined under test is parsed. The parsed contract has no errors reported. Unable to find the actual root cause as there is no stack trace available and finding the root cause is not taking any shape.

Is there any way to know what is causing the issue or probable causes which can cause the following error log.

**Comment by harikrishnan83** [comment]
@prateekkarmakar  Can you please share which version of Specmatic you are using, stacktrace of the error and your project setup (actuator library in pom.xml / build.gradle, application properties along with actuator setup if any). Thanks.

**Reply by prateekkarmakar** [reply]
@harikrishnan83 , using specmatic version 2.0.38 and spring boot actuator 2.7.18

**Comment by harikrishnan83** [comment]
@prateekkarmakar we have answered this point in [our response to one of your other queries](https://github.com/znsio/specmatic/discussions/1501#discussioncomment-11715051). Please have a look. Closing this issue since it is the same problem as #1501. Thanks.
